src/identity/identity-wallet.ts

```diff
@@ -539,13 +543,26 @@ export class IdentityWallet implements IIdentityWallet {
   }
 
   /** {@inheritDoc IIdentityWallet.issueCredential} */
-  async issueCredential(issuerDID: DID, req: CredentialRequest): Promise<W3CCredential> {
+  async issueCredential(
+    issuerDID: DID,
+    req: CredentialRequest,
+    opts?: MerklizerOptions
+  ): Promise<W3CCredential> {
     req.revocationOpts.id = req.revocationOpts.id.replace(/\/$/, '');
 
-    const schema = await new UniversalSchemaLoader('ipfs.io').load(req.credentialSchema);
-
-    const jsonSchema: JSONSchema = JSON.parse(byteDecoder.decode(schema));
+    let schema: object;
+    const loader =
+      opts?.documentLoader ??
+      getDocumentLoader({
+        ipfsNodeURL: 'ipfs.io'
```

Comment:
do not use ipfs.io -init with options from params if the document loader hasn't been passed.